README.md

```diff
@@ -24,7 +24,7 @@ This repository allows you to quickly install Apache Solr for Drupal 9 into a [D
 This is the classic Drupal solr:8 recipe used for a long time by Drupal users and compatible with search_api_solr. 
 
 * It installs a [`.ddev/docker-compose.solr.yaml`](docker-compose.solr.yaml) using the solr:8 docker image.
-* A standard Drupal 9 solr configuration is included in [.ddev/solr/conf](solr/conf).
+* A standard Drupal 9+ solr configuration is included in [.ddev/solr/conf](solr/conf).
```

Comment:
I think we should use the official/capitalized name for Solr s/solr/Solr

Reply:
This is still not capitalized

Reply:
Thanks, got it.

* You are welcome to do a PR to this to improve the README or anything else, but this PR was about using a different core name. 
* When you notice things like this, try to use the "suggest" option when you can, which makes it so the maintainer can just commit your suggestion.
<img width="838" alt="Allow_use_of_different_core_name__fixes__12_by_rfay_·_Pull_Request__14_·_drud_ddev-drupal9-solr" src="https://user-images.githubusercontent.com/112444/210138743-7c4add52-7ff5-4e0a-8e3e-d2a96ad635d6.png">

Reply:
@rfay, my remark was only for that occurrence, in the line that was already touched in this PR. I didn't suggest to replace all occurrences. Sorry for any trouble. But true, I'm often forgetting the "suggest" tool.